src/scripts/release.sh

```diff
@@ -25,6 +25,10 @@ fi
 if [ -n "$PARAM_GH_FILES" ]; then
 	set -- "$@" " $PARAM_GH_FILES"
 fi
+if [ -n "$PARAM_GH_ARGS" ]; then
+	set -- "$@" " $PARAM_GH_ARGS"
```

Comment:
If you double-quote `$PARAM_GH_ARGS`, Bash will treat its value as a string instead of breaking it down into expressions. 

Assuming `PARAM_GH_ARGS="--latest"`, the command will ultimately look like this:

```bash

gh release create my_tag my-file.sh "--latest"
```

Notice the quotes around the flag. You likely get an error similar to [this](https://github.com/CircleCI-Public/aws-ecr-orb/issues/253).

I would suggest the same [fix](https://github.com/CircleCI-Public/aws-ecr-orb/pull/254/files#diff-27420d37ce437f88af6c5acf291dbd88ecc4f48c6add4d18e44d99e4ab589941R76) used for the issue above:

```bash
# shellcheck disable=SC2086
gh release create "$PARAM_GH_TAG" \ 
 ${PARAM_GH_ARGS:+$PARAM_GH_ARGS} \
 "$@"
```

Alternatively, you can have the parameter alone, without the expansion, since it will just resolve to empty:

```bash
# shellcheck disable=SC2086
gh release create "$PARAM_GH_TAG" \ 
 $PARAM_GH_ARGS \
 "$@"
```